docs/UserGuide.md

```diff
@@ -544,18 +584,18 @@ Here are some potential error messages that you may receive and here's how to so
 
 Note: `yyyy-MM-dd` refers to the format of the date in the form of year-month-day. For example, 2023-11-01 refers to 1st November 2023.
 
-#### Find leave records by leave status: `find-leave-status`
+#### Find leave applications by leave status: `find-leave-status`
 
-To find leave records by leave status, use the `find-leave-status` command. This command will find leave records that have the specified status.
+To find leave applications by leave status, use the `find-leave-status` command. This command will find leave applications that have the specified status.
 
 1. Type in the following command in the command box:
-   `find-leave-status STATUS`, and replace `STATUS` with the status of the leave records you wish to view.
- - For instance: if you wanted to view all leave records that have been approved, you would type in `find-leave-status APPROVED` in the command box.
+   `find-leave-status STATUS`, and replace `STATUS` with the status of the leave applications you wish to view.
+ - For instance: if you wanted to view all leave applications that have been approved, you would type in `find-leave-status APPROVED` in the command box.
  - Do note that the status field is required, and the status must be either `APPROVED`, `PENDING` or `REJECTED`.
    - The status is case-sensitive. For instance, `approved` is not a valid status. Ensure you use the correct casing, which is `APPROVED`.
```

Comment:
I think the status might still be case-agnostic, at least based on the code i have with me